source/deployment/cluster.rst

```diff
@@ -1,116 +1,116 @@
-High Availability (E20) - Beta
+High Availability Cluster (E20)
 ===============================
 
-*Available in Enterprise Edition E20.*
+The high availability cluster feature enables a Mattermost system to maintain service during outages and hardware failures through the use of redundant infrastructure. It is available only in Enterprise Edition E20. 
```

Comment:
Let's keep *Available in Enterprise Edition E20* as a separate clear line below the title as the first item. 

@JeffSchering can we add something to style guide that Enterprise Edition features should have a note like this in documentation? We should eventually phase out the (E10), (E20) next to the title, as it doesn't work "(E10)" should technically be "(E10, E20)" and that breaks if we one day add a new SKU.